iroha2-kotlin-query-tool/src/main/kotlin/jp/co/genesis/service/IrohaService.kt

```diff
@@ -0,0 +1,136 @@
+package jp.co.genesis.service
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import jp.co.genesis.config.Iroha2Config
+import jp.co.genesis.model.Account
+import jp.co.genesis.model.AccountAsset
+import jp.co.genesis.model.AssetDefinition
+import jp.co.genesis.model.Domain
+import jp.co.genesis.model.PermissionToken
+import jp.co.genesis.model.Signature
+import jp.co.soramitsu.iroha2.asAccountId
+import jp.co.soramitsu.iroha2.client.Iroha2Client
+import jp.co.soramitsu.iroha2.generated.Mintable
+import jp.co.soramitsu.iroha2.privateKeyFromHex
+import jp.co.soramitsu.iroha2.publicKeyFromHex
+import jp.co.soramitsu.iroha2.query.QueryBuilder
+import jp.co.soramitsu.iroha2.toHex
+import kotlinx.coroutines.runBlocking
+import org.springframework.stereotype.Service
+import java.security.KeyPair
+
+@Service
+class IrohaService {
+
+    fun query(config: Iroha2Config): String {
+        if (config.url.isEmpty() || config.adminAccount.isEmpty() ||
+            config.adminPublicKeyHex.isEmpty() || config.adminPrivateKeyHex.isEmpty()
+        ) {
+            throw RuntimeException("Incorrect config")
```

Comment:
I'd separate this cases via `when` and throw exceptions with different messages

Reply:
done